packages/editor/src/plugins/image/components/pixabay-image-search.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { ChangeEvent, useState, useEffect } from 'react'
+
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+import { cn } from '@/helper/cn'
+
+interface PixabayResponse {
+  total: number
+  totalHits: number
+  hits: PixabayImage[]
+}
+interface PixabayImage {
+  id: number
+  webformatURL: string
+  tags: string
+  previewURL: string
+  largeImageURL: string
+}
+
+interface PixabayImageSearchProps {
+  onSelectImage: (imageUrl: string) => void
+}
+
+export const PixabayImageSearch = ({
+  onSelectImage,
+}: PixabayImageSearchProps) => {
+  const [query, setQuery] = useState('')
+  const [images, setImages] = useState<PixabayImage[]>([])
+  const [isLoadingImage, setIsLoadingImage] = useState(false)
+  const [isSearching, setIsSearching] = useState(false)
+
+  const [showTags, setShowTags] = useState(true)
+
+  const editorStrings = useEditorStrings()
+  const imageStrings = editorStrings.plugins.image
+  const apiKey = 'secret'
```

Comment:
this would be replaced by env?

Reply:
Correct

Reply:
why not do it now? less chance of someone like me accidentally pushing the key when I put it into a local env file

Reply:
Wasn't sure if the env is different for plugins or inside the editor package. Doing soon

Reply:
No idea how that works with integrations but if we use nextjs is should be available everywhere.

Reply:
Done